atomate/vasp/firetasks/run_calc.py

```diff
@@ -147,6 +148,11 @@ def run_task(self, fw_spec):
         elif job_type == "full_opt_run":
             jobs = VaspJob.full_opt_run(vasp_cmd, auto_npar=auto_npar, ediffg=self.get("ediffg"),
                                         max_steps=5, half_kpts_first_relax=False)
+        elif job_type == "neb":
```

Comment:
The job_type "neb" will call **VaspNEBJob**, which is used to handle the different file structures (for example, POSCARs are nested in subfolders).
Besides, all custodian arguments can be modified by the user now.